### ISSUE TITLE
[MOBILE]: Should buttons be full-width at mobile size for creating an Express Care appointment?

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#15086]
## Issue Description
While auditing the Express Care feature, I noticed some of the buttons were not full width at mobile size. Not sure this is how they should be positioned, or if buttons are meant to be full width at mobile size. Screenshots added for illustration below. Adding VSP design for a judgment call. 

---
## Tasks
- [ ] Confirm the design is correct, or offer guidance on updates needed

## Acceptance Criteria
- [ ] Design will be updated **or** ticket closed with no work needed

---
<img width="454" alt="Screen Shot 2020-10-19 at 5 40 34 PM" src="https://user-images.githubusercontent.com/934879/96646667-147c2d00-12f2-11eb-8e34-c93a3bc8b99d.png">

---
<img width="472" alt="Screen Shot 2020-10-19 at 5 40 45 PM" src="https://user-images.githubusercontent.com/934879/96646671-15ad5a00-12f2-11eb-8d65-15fc71bb4c92.png">

**Comment by rtwell** [comment]
update on this: the design system council has taken a look at this issue and will provide guidance asap. more than likely this will get tackled "back/next" buttons as well.

**Comment by outerpress** [comment]
@rtwell @emilywaggoner just checking in, any updates on this one?

**Comment by emilywaggoner** [comment]
Hey Carol, tagging you here as well because this is a design system issue -- I'm not sure where the Design System Council left off here but I'm moving it off of VAOS's plate because I think the Design System team should push it forward along with the council. Thank you!

**Comment by outerpress** [comment]
Created a[ new ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/419) for this to the VSP repo, closing here (cc: @laurenernest )
